omero_search_engine/__init__.py

```diff
@@ -33,6 +33,17 @@
 from configurations.configuration import app_config as config_
 
 
+def get_current_version():

```

Comment:
I found it strange to extract from changelog.
In other repositories we have used a version file. The version is then configure using bump2version

Reply:
Another option could be to retrieved and cached the version from the latest tag